usr/lib/pkcs11/ep11_stdll/ep11_func.h

```diff
@@ -197,9 +197,9 @@ typedef unsigned int (*m_admin_t) (unsigned char *response1, size_t * r1len,
                                    const unsigned char *cmd, size_t clen,
                                    const unsigned char *sigs, size_t slen,
                                    uint64_t target);
-typedef unsigned int (*m_add_backend_t) (const char *name, unsigned int port);
-typedef unsigned int (*m_init_t) (void);
-typedef unsigned int (*m_shutdown_t) (void);
+typedef int (*m_add_backend_t) (const char *name, unsigned int port);
+typedef int (*m_init_t) (void);
+typedef int (*m_shutdown_t) (void);
```

Comment:
These typedefs are derived from the EP11 library API functions. They must match those from the EP11 library. Are you sure that the EP11 library uses int?

Reply:
The function prototypes in ep11.h have return type int. I assumed the header to be a copy of the original ep11 header file and thus decisive.

Moreover, m_init return value is checked for < 0 to indicate init failure, meaning m_init would always succeed if it returned an unsigned value.